content/swagger/resources/measurement_families/routes/measurement_families.yaml

```
@@ -0,0 +1,165 @@
+get:
+  summary: Get list of measurement families
+  operationId: "measurement_families_get_list"
+  description: This endpoint allows you to get a list of measurement families. Measurement families are paginated and sorted by code.
+  tags:
+    - Measurement family
+  x-versions:
+    - "Serenity"
+  responses:
+    200:
+      description: Return measurement families paginated
+      schema:
+        title: Measurement Families
+        type: object
+        allOf:
+          - $ref: '#/definitions/Pagination'
+          - properties:
+              _embedded:
+                type: object
+                properties:
+                  items:
+                    type: array
+                    items:
+                      $ref: '#/definitions/MeasurementFamilyList'
+      examples: {
+        _links: {
+            "self": {
+              "href": "https://demo.akeneo.com/api/rest/v1/measurement-families?page=1&limit=1"
+            },
+            "first": {
+              "href": "https://demo.akeneo.com/api/rest/v1/measurement-families?page=1&limit=1"
+            }
+          },
+        "current_page": 1,
+        _embedded: {
+          "items": [
```

```suggestion
      examples: [
```
No need for pagination as there won't be more than 100 measurement families.